src/main/webapp/src/app/models/hidden-gem.ts

```diff
@@ -0,0 +1,18 @@
+import { Geometry } from './geometry';
+import { Photos } from './photos';
+import { OpeningHours } from './openingHours';
+
+export interface HiddenGem {
+  geometry: Geometry;
```

Comment:
same here, it would be good to have a short little comment for each element explaining what is being stored in each variable, even if it seems obvious.

... and the other two model files.